modules/cms/classes/Controller.php

```diff
@@ -488,11 +488,13 @@ protected function initTwigEnvironment()
 
         $useCache = !Config::get('cms.twigNoCache');
         $isDebugMode = Config::get('app.debug', false);
+        $strictVariables = (Config::get('cms.enableTwigStrictVariables', null) !== null) ?: $isDebugMode;
         $forceBytecode = Config::get('cms.forceBytecodeInvalidation', false);
 
         $options = [
             'auto_reload' => true,
             'debug' => $isDebugMode,
+            'strict_variables' => $isDebugMode,
```

Comment:
This needs to be `$strictVariables`, please test each config value and ensure that they're working as intended (true = enabled, false = disabled, null = value of app.debug)

Reply:
Ouch, a bad copy&paste from my testing installation :sweat:
I made all config combination test with `dd()` just after var assignment.